includes/testing-library.sh

```diff
@@ -120,6 +155,8 @@ clean_up() {
     fi
 
     docker rm -f -v selenium-for-tests
+    # Remove any subprocesses. For example the drush webserver process
+    pkill -P $$
```

Comment:
This will not work if the script is not run in one go

Reply:
Let's handle this in a separate issue then.